dir.props

```diff
@@ -21,6 +19,10 @@
     <CliVersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionPatch)-$(ReleaseSuffix)</CliVersionPrefix>
 
     <CliTargetFramework>netcoreapp2.0</CliTargetFramework>
-
   </PropertyGroup>
+
+  <Import Project="build/Microsoft.DotNet.Cli.InitRepo.props" />
+  <Import Condition=" Exists('$(GitCommitInfoProps)') Or '$(GeneratingPropsFile)' != 'true' " Project="$(GitCommitInfoProps)" />
```

Comment:
I need some help. This line is failing because the condition is evaluating to true when it shouldn't. Here's the strange thing. Run this:
dotnet msbuild build.proj /p:Architecture=x64/p:GeneratingPropsFile=true /t:WriteDynamicPropsToStaticPropsFiles 

And it fails. Now remove the two import lines and run it again. It passes. Now delete the artifacts directory and run it again. It passes.

So what is different? The difference is somehow it's passing if the build_projects have been built already. If you now go and delete those obj and bin files and try again it will fail. Why? Why does that affect the value of my property GeneratingPropsFile?